### ISSUE TITLE
asapdiscovery-modeling README and scripts/ folder

### ISSUE THREAD

**Issue by mariacm12** [choderalab/asapdiscovery#852]
I'm looking at the `asapdiscovery-modeling` README file and a lot of the files being referenced are missing (mainly from the `scripts/` folder). For example, `./asapdiscovery/modeling/scripts/copy_and_clean_frag.sh` is missing. Was the repo and workflow updated at some point and the README wasn't updated?

**Comment by hmacdope** [comment]
Yes, many times over! We have gone through a lot of iterations on scripts, API and design. If you spot anything like this just raise an issue (as you have done :+1: ). Feel free to also PR fixing the incorrect docs.

**Comment by apayne97** [comment]
That whole README in particular is pretty out of date bc we haven't done much with colabfold since those initial efforts
